app/graphql/mutations/users/delete_user.rb

```diff
@@ -4,7 +4,11 @@ class Mutations::Users::DeleteUser < GraphQL::Schema::Mutation
   type Types::UserType
 
   def resolve(attributes)
-    user = User.find(attributes[:id])
-    user.destroy
+    user = User.find_by(id: attributes[:id])
+    if user
```

Comment:
I really like this conditional here. I imagine it avoids breaking other methods if user is nil?